pkgs/development/haskell-modules/configuration-ghc-9.6.x.nix

```diff
@@ -108,6 +108,11 @@ self: super: {
   # Jailbreaks for servant <0.20
   servant-lucid = doJailbreak super.servant-lucid;
 
+  # support for transformers >= 0.6
+  fused-effects-random = doJailbreak super.fused-effects-random;
+  fused-effects-readline = doJailbreak super.fused-effects-readline;
+  fused-effects-exceptions = doJailbreak super.fused-effects-exceptions;
+
```

Comment:
Is there a reason not to put this in configuration-common.nix? I guess it would also be needed in future ghc versions.

Reply:
Because transformers is shipped with GHC I thought it wouldn’t make sense to put it for GHC <6

Reply:
Yeah, that is true. This would be nicer if we finally had a more specific allowNewer function.

I think the work saved by putting this into configuration-common for future ghc versions outways the risk of erroneous jailbreaks on older ghcs.